docs/user/managing-devices.md

```diff
@@ -594,3 +594,10 @@ flightctl console <some_device_name>
 To disconnect, enter "exit" on the console. To force-disconnect, press `<ctrl>+b` three times.
 
 ## Decommissioning Devices
+
+## Using Image Pull Secrets
+
+If your device relies on containers from a private repository, authentication credentials (pull secrets) must be placed in the appropriate system paths.
+
+* **OS Image:** Uses `/etc/ostree/auth.json`
+* **Container Images:** Uses the system default for Podman, `/etc/containers/auth.json`
```

Comment:
I'm wondering whether this (also) belongs into the "[Building the OS Image](https://github.com/flightctl/flightctl/blob/main/docs/user/building-images.md#building-the-os-image-bootc)" docs, e.g. as a note below the Containerfile?
And in this doc we might want to move it to a more prominent place, maybe the managing OS updates section?